### PR TITLE
chore(docs): update url from fb.me to reactjs.org 

### LINKED ISSUE #16109: Consider using reactjs.org instead of fb.me in react's error/warning messages
Hi,

Facebook is blocked in my country (and I think in some other countries) and links like `https://fb.me/react-invalid-hook-call` are blocked for me, in order to see the related documentation of the link I have to use VPN. so if is possible please use another link shortener service or point to `reactjs.org` directly.

Thanks.

### PR DESCRIPTION
**Pull request by adnaanbheda**
Fixes #16109

There was a single link left out in https://github.com/facebook/react/pull/19598, I've updated the url and removed a comment to reflect that the change is complete.

### CODE REVIEW

File: packages/react/src/__tests__/createReactClassIntegration-test.js [hunk 1]
```diff
@@ -365,7 +365,7 @@ describe('create-react-class-integration', () => {
       expect(() => Component()).toThrow(),
     ).toErrorDev(
       'Warning: Something is calling a React component directly. Use a ' +
-        'factory or JSX instead. See: https://fb.me/react-legacyfactory',
+        'factory or JSX instead. See: https://reactjs.org/warnings/legacy-factories.html',
```
Comment (bvaughn) [inline packages/react/src/__tests__/createReactClassIntegration-test.js:368]: This change causes the test to fail, because the `create-react-class` package has not been changed. (Indeed that class has not been updated in 3 years.)

I think it's fine to leave the fb.me reference in this integration test. Let's revert the change to this file?

File: packages/scheduler/src/forks/SchedulerHostConfig.default.js [hunk 2]
```diff
@@ -85,7 +85,7 @@ if (
     // option to rely on it in the future?
     const requestAnimationFrame = window.requestAnimationFrame;
     const cancelAnimationFrame = window.cancelAnimationFrame;
-    // TODO: Remove fb.me link
```
Comment (bvaughn) [inline packages/scheduler/src/forks/SchedulerHostConfig.default.js:88]: This comment looks good to remove. The links it referred to (also added in #14964) were removed in #19598 so it's fine to clean the comment up.

**Review by bvaughn — CHANGES_REQUESTED** [review]

File: packages/react/src/__tests__/createReactClassIntegration-test.js (hunk 1 above)
Comment (adnaanbheda) [reply packages/react/src/__tests__/createReactClassIntegration-test.js:368]: @bvaughn I'm reverting the change but I don't understand how is an error string causing the test to fail.
How can we approach this ?

**Review by bvaughn — APPROVED** [review]
